R/estimate_RIT.R

```diff
@@ -0,0 +1,103 @@
+#' @title estimate RIT score for a student using regression or interpolation
+#' 
+#' @description Given studentid, measurementscale, and a target_date, the 
+#' function will return an estimated score based on selected method 
+#' 
+#' @param mapvizieR_obj mapvizieR object
+#' @param sid target studentid
+#' @param measurescale target subject
+#' @param target_date date of interest, %Y-%m-%d format
+#' @param method which method to use to estimate RIT score
+#' @param foward default is TRUE, set to FALSE if only scores before target_date should be chosen for 'closest' method
+#' 
+#' @export
+
+
+estimate_rit <- function(
+  mapvizieR_obj,
+  sid,
```

Comment:
this is minor, but can we can this parameter `studentid`?  that's what we've been using elsewhere.